docs/modules/ROOT/pages/how-tos/logging/increase-elasticsearch-storage-size.adoc

```diff
@@ -0,0 +1,283 @@
+
+. Stop operator from managing the cluster
++
+[source,bash]
+----
+kubectl \
+  --as=cluster-admin \
+  -n openshift-logging \
+  patch clusterloggings/instance \
+  --type=merge \
+  -p '{"spec":{"managementState":"Unmanaged"}}'
+----
++
+. Scale down Fluentd pods to stop sending logs to Elasticsearch
++
+[source,bash]
+----
+kubectl \
+  --as=cluster-admin \
+  -n openshift-logging \
+  patch daemonset fluentd \
+  -p '{"spec":{"template":{"spec":{"nodeSelector":{"logging-infra-fluentd": "false"}}}}}'
+----
++
+. Perform a shard synced flush using the `es_util` tool to ensure there are no pending operations waiting to be written to disk prior to shutting down.
++
+[source,bash]
+----
+es_pod=$(kubectl \
+  --as=cluster-admin \
+  -n openshift-logging \
+  get pods \
+  -l component=elasticsearch \
+  -o name | head -n1)
+
+kubectl \
+  --as=cluster-admin \
+  -n openshift-logging \
+  exec "${es_pod}" \
+  -c elasticsearch \
+  -- es_util --query="_flush/synced" -XPOST
+----
++
+Example output:
++
+[source,json]
+----
+{"_shards":{"total":4,"successful":4,"failed":0},".security":{"total":2,"successful":2,"failed":0},".kibana_1":{"total":2,"successful":2,"failed":0}}
+----
++
+. Prevent shard balancing when purposely bringing down nodes.
++
+[source,bash]
+----
+kubectl \
+  --as=cluster-admin \
+  -n openshift-logging \
+  exec "${es_pods[1]}" \
+  -c elasticsearch \
+  -- es_util --query="_cluster/settings" -XPUT -d '{ "persistent": { "cluster.routing.allocation.enable" : "primaries" } }'
+----
++
+Example output:
++
+[source,json]
+----
+{"acknowledged":true,"persistent":{"cluster":{"routing":{"allocation":{"enable":"primaries"}}}},"transient":{}}
+----
++
+. Find Elasticsearch deployments
++
+[source,bash]
+----
+kubectl \
+  --as=cluster-admin \
+  -n openshift-logging \
+  get deploy \
+  -l component=elasticsearch
+----
++
+Sample output:
++
+[source]
+----
+NAME                           READY   UP-TO-DATE   AVAILABLE   AGE
+elasticsearch-cdm-7ya69va8-1   1/1     1            1           68d
+elasticsearch-cdm-7ya69va8-2   1/1     1            1           68d
+elasticsearch-cdm-7ya69va8-3   1/1     1            1           68d
+----
++
+. For each deployment do
+.. Restart Elasticsearch
++
+[source,bash]
+----
+ES_DEPLOYMENT=elasticsearch-cdm-7ya69va8-1 <1>
+
+kubectl \
+  --as=cluster-admin \
+  -n openshift-logging \
+  scale deploy/${ES_DEPLOYMENT} \
+  --replicas=0
+
+# Verify pod is removed
+kubectl \
+  --as=cluster-admin \
+  -n openshift-logging \
+  get pods \
+  | grep "${ES_DEPLOYMENT}-"
+
+kubectl \
+  --as=cluster-admin \
+  -n openshift-logging \
+  scale deploy/${ES_DEPLOYMENT} \
+  --replicas=1
+
+# Wait for pod to become ready
+kubectl \
+  --as=cluster-admin \
+  -n openshift-logging \
+  get pods \
+  --watch
+----
+<1> Replace with deployment name found in previous step.
++
+.. Wait until cluster becomes healthy again.
++
+[WARNING]
+Make sure the status is `green` or `yellow` before proceeding.
```

Comment:
I think we should decide on which "color", this is unclear
```suggestion
Make sure the status is at least `yellow` before proceeding.
```

Reply:
I thing it's clearer if both possible colors are explicitly mentioned. For me the cluster stayed green while restarting two of three nodes.